modules/concept-docs/pages/analytics-for-sdk-users.adoc

```diff
@@ -3,6 +3,8 @@
 :page-topic-type: concept
 :page-aliases: analytics,
 
+include::partial$attributes.adoc[]
```

Comment:
Doesn't this need to be
include::project-docs:partial$attributes.adoc[]
?

Reply:
This will work fine, and the work of removing the placeholder pointer attributes.adoc is ticketed, so I think no point doing partial extra work now?

Reply:
Ah, I thought Scala had been done - 
looks like it's waiting for @programmatix to approve https://github.com/couchbase/docs-sdk-scala/pull/34
and hence that 1.1 change didn't make it to 1.2    :face-palm: